RELEASE-NOTES.md

```diff
@@ -1,5 +1,8 @@
 ## RELEASE NOTES
 
+### Version 2.65.7
```

Comment:
@paulhowes-HMCTS - there appears to be a mismatch in version numbers... you've got `2.65.7` here but `2.66.0` in `package.json`?

Reply:
I've pushed a new version of the release notes up now.